packages/frontend/src/pages/Register.tsx

```diff
@@ -118,6 +119,24 @@ export default function Register() {
             <FormHelperText>Enter your password.</FormHelperText>
           )}
         </FormControl>
+        <FormControl isInvalid={!!errors.sex } isRequired>
```

Comment:
@sebasptsch Not being enforced in form control from quick check, pls check for regressions here. Excluding this LGTM